src/ocamlorg_frontend/components/learn_tutorial_block.eml

```diff
@@ -0,0 +1,53 @@
+let tutorial_block_content 
+~title
+~heading
+~description
+~(tutorial_links:string list)
```

Comment:
this type will have to be a bit different, something like
```
type link = { href: string; title: string }
```

`tutorial_links: link list`